scripts/setup

```diff
@@ -98,7 +99,7 @@ if [ "${IS_VIRTUALENV}" != 1 ]; then
 fi
 
 # options may be followed by one colon to indicate they have a required argument
-if ! options=$(getopt -o dsbwm -l -t deployment,create_items_holdings_small,create_items_holdings_big,lazy,pursue,warnings,data_path,time,es-mapping: -- "$@")
+if ! options=$(getopt -o dPsbclptmwD: -l -t deployment,persons,create_items_holdings_small,create_items_holdings_big,lazy,pursue,time,es-mapping,warnings,data_path: -- "$@")
```

Comment:
I think we can change `-l -t` to `-l` as `-t` doesn't exists as argument in `getopt` command. What do you think?

Reply:
Will try, can not find any help pages for this.